homeassistant/core.py

```diff
@@ -1273,8 +1260,6 @@ def __init__(
         self.context = context or Context()
         self.domain, self.object_id = split_entity_id(self.entity_id)
         self._as_dict: ReadOnlyDict[str, Collection[Any]] | None = None
```

Comment:
Its too bad that we can't get rid of this one but the json helper calls `as_dict()` not `.as_dict`

Reply:
We could teach it about `as_dict_json` and make it a non-callable. Or we could check if `as_dict` is a callable, and only call it if that's the case.

Reply:
good idea. Will do that it it comes up in the profile again